cpp/src/plasma/plasma.h

```diff
@@ -51,6 +53,12 @@
     }                                                                       \
   } while (0);
 
+/// Fix visibility of the Python C extension entry point
+#ifdef PyMODINIT_FUNC
+#undef PyMODINIT_FUNC
+#endif
+#define PyMODINIT_FUNC extern "C" __attribute__((visibility ("default"))) PyObject*
```

Comment:
This is very odd, will have to have a closer look at what is going on; I've never had any issues like this